README.md

```diff
@@ -222,22 +236,256 @@ This function will also automatically add the `crossorigin` attribute for fonts,
+  }
+);
+```
+
+### Removing a Symbol
+
+Use `am_deregister_symbol` to remove a registered a symbol.
+
+This should be added via an action that fires after, or at a lower priority, than the action used for `am_register_symbol`.
+
+```php
+am_deregister_symbol( $handle = '' );
+```
+
+**`$handle`**
+
+> `string`
+> 
+> The handle with which the symbol was registered.
+
+**Return**
+
+> `bool`
+>
+> Whether the symbol has been deregistered and removed from the sprite. `true` on success, or if the symbol hadn't been previously registered; `false` on failure.
+
+#### Replacing a symbol
+
+Prior to re-registering a symbol, verify the symbol to be replaced is not registered.
+
+```php
+if ( am_deregister_symbol( 'logomark' ) ) {
+  am_register_symbol(
+    [
+      'handle'    => 'logomark',
+      'src'       => 'svg/logomark-alt.svg',
+      'condition' => 'global',
+    ]
+  );
+}
+```
+
+### Displaying a Symbol
+
+`am_use_symbol` prints an `<svg>` element with the specified attributes.
+
+```php
+am_use_symbol( $handle = '', $attributes = [] );
+```
+
+**`$handle`**
+
+> `string`
+> 
+> The handle with which the symbol was registered.
+
+**`$attributes`**
+
+> `array` 
+> 
+> An array of attribute-value pairs to add to the resulting SVG markup.
+> 
+> Override global attributes, or those defined via `am_register_symbol`, by declaring a new value here; remove it entirely by passing a falsy value.
+
+#### Notes on SVG sizing
+
+Asset Manager will attempt to establish a default size for each SVG, which will be used to calculate the dimensions if only one, or neither, of `height` or `width` is passed to `am_use_symbol`.
+
+The default size is based on (in order):
+1. The values set in the symbol's `am_register_symbol` attributes array
+1. The `height` and `width` attributes from the SVG
+1. The `viewBox` attribute values
+
+If Asset Manager cannot determine a symbol's dimensions, both `height` _and_ `width` will need to be declared in the `attributes` array passed to `am_use_symbol`.
+
+**The simplest way to ensure SVGs are sized as expected is to verify each file's `<svg>` element either has both `height` and `width` attributes or a viewBox attribute.**
+
+_**Example**_:
+
+```php
+am_use_symbol(
+  'logomark',
+  [
+    'width' => 200,
+    'class' => 'header-logo',
+  ]
+);
+```
+
+_**Output**_:
+
+```html
+<svg width="200" height="27.67" class="header-logo" aria-hidden="true" focusable="false">
+  <use href="#am-symbol-logomark"></use>
+</svg>
+```
+
+### Getting a Symbol
+
+`am_get_symbol` returns a string containing the `<svg>` element with the specified attributes.
+
+```php
+$symbol_markup = am_get_symbol( $handle = '', $attributes = [] );
+```
+
+This function uses the same arguments as `am_use_symbol`.
+
+_**Example**_:
+
+```php
+$logomark_svg_markup = am_get_symbol(
+  'logomark',
+  [
+    'width' => 200,
+    'class' => 'header-logo',
+  ]
+);
+```
 
 ## Requirements
 
-* WordPress: 4.7+
-* PHP: 7.0+
+* WordPress: 5.2.0+
+* PHP: 7.1+
```

Comment:
I'd argue to bump to 7.4 for our sanity and to mirror what we're testing against in CI.